governance/ProductDeprecation.md

```diff
@@ -0,0 +1,26 @@
+# Product Deprecation
```

Comment:
Should this be called FeatureDeprecation and not ProductDeprecation?

Reply:
Good call, I agree, this is something the TSC would need to agree on and then each sig would need to monitor and/or verify this is happening correctly in their areas

Reply:
Okay understood, I wasn't sure the best sig for this but as deprecation stages coincide with releases this seemed like a reasonable place to start. I can move these new docs to the sig-core repo instead if that would be preferable? Also is there a list to loop in the TSC?

I can update the name to Feature Deprecation as well (we just want something to disambiguate macro vs micro deprecations if that makes sense)

Reply:
TSC doesn't have a repo, so leaving it here seems okay for now.  You would need to get it added to the TSC agenda for their next meeting to discuss/agree/etc.

Reply:
Okay understood, I've pinged @lawsonamzn about this and asked if this can be added to the next TSC agenda.

Reply:
@jeremyong  is actually the chair of the TSC, and yeah, this should be added.

Reply:
Added to tomorrow's agenda here: https://github.com/o3de/tsc/issues/4

Reply:
Awesome thank you, it'd be great to know how best to proceed with this PR (someone else is welcome to take ownership of it or we can throw it out and start over, but it would be great to know how to handle deprecations one way of the other 🙂)